x-pack/plugins/observability/public/components/threshold/components/alert_flyout.tsx

```diff
@@ -28,7 +28,7 @@ export function AlertFlyout(props: Props) {
     () =>
       triggersActionsUI &&
       triggersActionsUI.getAddRuleFlyout({
-        consumer: 'infrastructure',
+        consumer: 'alerts',
```

Comment:
Is that the consumer id for the threshold rule? where did you define this?

Reply:
Yes. I don't think we define that on the rule creation. But the consumer tells where the rule is created from, and it helps to handle the authorization.